test/common/tests.go

```diff
@@ -30,5 +30,8 @@ var (
 		{"B05 - Verify Codeready CRUDL permissions", TestCodereadyCrudlPermisssions},
 		{"A09 - Verify Subscription Install Plan Strategy", TestSubscriptionInstallPlanType},
 		{"B06 - Verify users with no email get default email", TestDefaultUserEmail},
+		{"F05 - Verify Replicas Scale correctly in Threescale", TestReplicasInThreescale},
+		{"F06 - Verify Replicas Scale correctly in Apicurito", TestReplicasInApicurito},
+		{"F07 - Verify Replicas Scale correctly in Amq Streams", TestReplicasInAmqstreams},
```

Comment:
@CathalOConnorRH You are correct by the looks of it. The E2E tests are only checking the managed type. This is a larger issue to resolve and should be outside of this PR @philbrookes in my opinion.

For now, I would remove this line here so the AMQ Streams test does not get run and open a new JIRA please for adding the ability to support different installation types in the e2e tests.

Reply:
I've updated that now and created a ticket 
https://issues.redhat.com/browse/INTLY-6767

Reply:
Thanks @CathalOConnorRH . e2e tests should pass now. 

@philbrookes Are you ok with this path forward?